module1/person.go

```diff
@@ -0,0 +1,44 @@
+package module1
+
+import "fmt"
+
+type Person struct {
+	Name         string
+	Surname      string
+	Relationship string
+	Age          int
+	Hobbies      []string
+}
+
+func AllocNewPersonPtr(
+	name, surname, relationship string, age int,
+	hobbies []string) *Person {
+	return &Person{
+		Name:         name,
+		Surname:      surname,
+		Relationship: relationship,
+		Age:          age,
+		Hobbies:      hobbies,
+	}
+}
+
+func (p Person) Greet() string {
+	greeting := fmt.Sprintf("Hello, my name is %s!", p.Name)
+	return greeting
+}
+
+func (p Person) IsFamily() (bool, string) {
+	possibleRelatives := []string{"dad", "mom", "grandpa", "grandma", "granny", "brother", "sister"}
+
+	for _, relative := range possibleRelatives {
```

Comment:
Use index, here useless copy)
```suggestion
	for i := range possibleRelatives {
```
When we write `for _, relative := range possibleRelatives` from `possibleRelatives` all objects copy to `relative`